### PR TITLE
fix: completion test log issue

### PR DESCRIPTION
**Pull request by pratikjagrut**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind bugfix

**What does this pull request do? Which issues does it resolve?** (use `resolves #<issue_number>` if possible) 
resolves #
#1864

### CODE REVIEW

File: cmd/run.go [hunk 1]
```diff
@@ -201,12 +201,25 @@ func (cmd *RunCmd) RunRun(f factory.Factory, args []string) error {
...
 	// Set config root
 	configLoader := f.NewConfigLoader("")
-	configExists, err := configLoader.SetDevSpaceRoot(logger)
+	configExists, err := configLoader.SetDevSpaceRoot(log.Discard)
 	if err != nil {
 		return nil, err
 	}
+
+	// set working dir back to original
+	err = os.Chdir(cwd)
```
Comment (FabianKramm) [inline cmd/run.go:218]: please put this as an defer that's executed on function exit

**Review by FabianKramm — CHANGES_REQUESTED** [review]
@pratikjagrut thanks for the PR! Please take a look at my comments

File: cmd/run.go (hunk 1 above)
Comment (pratikjagrut) [reply cmd/run.go:218]: okay

**Review by FabianKramm — APPROVED** [review]
LGTM!

**Pull request merged**